app/models/document.rb

```diff
@@ -75,6 +86,18 @@ def published?
     self.class.for_publication.where(id: id).any?
   end
 
+  def created_by
+    if user.present?
+      "This document was manually uploaded by #{user.name}"
+    elsif api_user.present?
+      "This document was uploaded by #{api_user.name}"
+    end
+  end
+
+  def date_received_or_created
+    date_received.present? ? date_received.to_date.strftime("%e %B %Y") : created_at.to_date.strftime("%e %B %Y")
```

Comment:
shouldn't this be using our `day_month_year` format?

also I'd write `(date_received || created_at).to_date.to_formatted_s(:day_month_year)`